dot-net-sdk/validators/BanditEvaluator.cs

```diff
@@ -105,7 +105,9 @@ public BanditEvaluator(int totalShards = 10_000)
     {
         var numberOfActions = actionScores.Count;
 
-        var bestAction = actionScores.Aggregate((currentmax, next) => currentmax.Value > next.Value ? currentmax : next);
+        // var bestAction = actionScores.Aggregate((currentmax, next) => currentmax.Value > next.Value ? currentmax : next);
+        // Order by key then by value to get the highest score, tie broken by action key.
+        var bestAction = actionScores.ToList().OrderByDescending(action=>action.Value).ThenBy(action=>action.Key).First();
```

Comment:
What's the magnitude of the list of actions? The double sort is not the most efficient but it reads so nicely. Aggregating is O(1) but not as pretty to look at and understand.

Reply:
I imagine most lists of actions will be 10-100.

Note that ChatGPT says `OrderByDescending()` chained with `ThenBy()` will still result in a single-pass sort

Reply:
I suspected that may be the case. thanks!